README.md

```diff
@@ -416,6 +416,19 @@ const result = trimStart(str)
 
 ## Strongly-typed alternatives to common loosely-typed functions
 
+### lowerCase
+
+This function converts a string to `lower case` at both runtime and type levels.
+_NOTE: this function will split by words and join them with `" "`._
```

Comment:
Suggestion for a better warning?
I want it to be crystal clear that `toUpperCase` and `upperCase` are significantly different
(Same with the lower counterparts)